.prettierignore

```diff
@@ -0,0 +1,40 @@
+# Auto-generated files
```

Comment:
Ignore auto-generated files or anything that could potentially cause slow prettier runs